internal/driver/glfw/loop.go

```diff
@@ -112,74 +114,81 @@ func (d *gLDriver) runGL() {
 	if d.trayStart != nil {
 		d.trayStart()
 	}
+
 	fyne.CurrentApp().Lifecycle().(*app.Lifecycle).TriggerStarted()
-	for {
-		select {
-		case <-d.done:
-			eventTick.Stop()
-			d.drawDone <- nil // wait for draw thread to stop
+
+	// Post empty event to not get stuck in the loop on first iteration.
+	PostEmptyEvent()
+
+	for range eventTick.C {
```

Comment:
Do we even need `eventTick` anymore? Can this just be a "forever" loop now that we are blocking with `d.waitForEvents()`?

Reply:
My idea for keeping it was if we got more inputs that 60 times per second but you might be right.